content/touchpad.md

```diff
@@ -20,9 +20,15 @@ tableOfContents: true
 
 Press <kbd>Fn</kbd>+<kbd>F1</kbd> to turn your laptop touchpad on/off.
 
+>**Note**: This shortcut is <kbd>Fn</kbd>+<kbd>F6</kbd> on the Pangolin (pang12).
+
 To configure your touchpad, press the Super Key <kbd><font-awesome-icon :icon="['fab', 'ubuntu']"></font-awesome-icon></kbd>, <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon></kbd> then type <u>Mouse & Touchpad</u> and click on the result. The touchpad settings are under the **Touchpad** tab.
 
-![Mouse Search](/images/touchpad/mouse.png)
+![Mouse Search](/images/touchpad/touchpad-mouse.png)
+
+From here the touchpad can be disabled only while typing like in the screenshot below:
```

Comment:
> From here the touchpad can be disabled only while typing like in the screenshot below:

A comma should go after `From here`, and also before `like`.

Reply:
Should be fixed in latest commit.